plugins/woocommerce/client/blocks/assets/js/base/components/drawer/utils/aria-helper.ts

```diff
@@ -13,9 +16,14 @@ let hiddenElements: Element[] = [],
 	isHidden = false;
 
 /**
- * Determines if the passed element should not be hidden from screen readers.
+ * Determines if the passed element should be hidden from screen readers.
  *
- * @param {HTMLElement} element The element that should be checked.
+ * An element will be hidden unless it is one of the following:
+ * - a <script> tag
+ * - has `aria-hidden`
+ * - has `aria-live`
+ * - has `data-keep-visible`
+ * - has a live region role (`alert`, `status`, `log`, `marquee`, or `timer`)
  *
  * @return {boolean} Whether the element should not be hidden from screen-readers.
```

Comment:
We need to update it here too: :slightly_smiling_face: 

```suggestion
 * @return {boolean} Whether the element should be hidden from screen-readers.
```